Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -91,6 +91,18 @@ public float NearObjectAxisLerp
             set => nearObjectAxisLerp = value;
         }
 
+        [SerializeField]
+        [Min(0.0f)]
+        [Tooltip("Smoothing factor for near object detection sensitivity")]
```

Comment:
Please explain what the smoothing factor does, so that somebody can understand its behavior without needing to read the code. I liked the expression you used in your PR descriptions.